python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -746,11 +803,11 @@ def _is_transient_stop_reason(self, stopped_reason: str):
 
         return False
 
-    def _is_transient_startup_failure(self, run, task):
-        if not task.get("stoppedReason"):
+    def _is_transient_startup_failure(self, run: DagsterRun, task: Dict[str, Any]):
+        if task.get("stoppedReason") is None:
             return False
         return run.status == DagsterRunStatus.STARTING and self._is_transient_stop_reason(
-            task.get("stoppedReason")
+            task.get("stoppedReason", "")
```

Comment:
This isn't entirely necessary, I was mostly just trying to put type-hints in wherever they were missing and needed to do some gymnastics to get pyright to be happy about it